### PR TITLE
Updates to configuring profiles and settings

### PR DESCRIPTION
**Pull request by cicdw**
This PR updates the configuration of profiles and settings and greatly compresses the content into a more streamlined and hopefully digestible form.

Additionally, this adds a `-y` flag for unsetting values from an active profile.

### CODE REVIEW

**Review by billpalombi — COMMENTED** [review]
A few tweaks

It looks like the staging build in't working but its not obvious why that is

File: docs/3.0rc/manage/settings-and-profiles.mdx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+### Prefect Server
+
+- **`PREFECT_HOME`**: the `PREFECT_HOME` value specifies the local Prefect directory for configuration files,
+profiles, and the location of the default Prefect SQLite database.
+- **`PREFECT_API_DATABASE_CONNECTION_URL`**: the database connection URL for a locally hosted Prefect instance.
+Must be provided in a SQLAlchemy-compatible format. Prefect currently supports SQLite and Postgres. 
+
+## Security settings
+
+### Host the Prefect UI behind a reverse proxy
```
Comment (billpalombi) [inline docs/3.0rc/manage/settings-and-profiles.mdx:113]:
```suggestion
### Host the UI behind a reverse proxy
```

**Review by billpalombi — APPROVED** [review]
One remaining nit but otherwise lgtm. Build works now and looks good.

**Comment by cicdw** [comment]
Awesome, thanks for the thorough review @billpalombi !

File: src/prefect/cli/config.py [hunk 2]
```diff
@@ -118,8 +118,12 @@ def unset(settings: List[str]):
...
-    if is_interactive() and not typer.confirm(
-        f"Are you sure you want to unset the following settings: {settings!r}?",
+    if (
+        not confirm
+        and is_interactive()
+        and not typer.confirm(
+            f"Are you sure you want to unset the following settings: {settings!r}?",
+        )
     ):
         exit_with_error("Unset aborted.")
```
Comment (zzstoatzz) [inline src/prefect/cli/config.py:128]: ✨

**Review by zzstoatzz — APPROVED** [review]

**Pull request merged**